packages/apps/src/configs/wallets/currency-config.ts

```diff
@@ -0,0 +1,31 @@
+import EdgwareLogo from '@webb-dapp/apps/configs/wallets/logos/EdgwareLogo';
+import EtherLogo from '@webb-dapp/apps/configs/wallets/logos/Eth';
+import { AppConfig } from '@webb-dapp/react-environment';
+import React from 'react';
+
+export const currenciesConfig: AppConfig['currencies'] = {
+  1: {
+    name: 'Edgware token',
+    symbol: 'EDG',
+    color: '',
+    id: 1,
+    supportingChainsIds: [3, 1],
```

Comment:
Chains already know the IDs of currencies, why do currencies need to know the chains which support them?

Reply:
for easier access, we can just do this on the class for `Currency` too , from chains config we get the supported chains from a currency

Reply:
YOu have a good point BTW will remove it In the meantime and add implementation for this on the Currency